src/xinterpreter.cpp

```diff
@@ -73,31 +73,20 @@ namespace xcpp
         xeus::register_interpreter(this);
     }
 
-    static std::string get_stdopt()
+    static std::string get_stdopt(int argc, const char* const* argv)
     {
-        // We need to find what's the C++ version the interpreter runs with.
-        const char* code = R"(
-int __get_cxx_version () {
-#if __cplusplus > 202302L
-    return 26;
-#elif __cplusplus > 202002L
-    return 23;
-#elif __cplusplus >  201703L
-    return 20;
-#elif __cplusplus > 201402L
-    return 17;
-#elif __cplusplus > 201103L || (defined(_WIN32) && _MSC_VER >= 1900)
-    return 14;
-#elif __cplusplus >= 201103L
-   return 11;
-#else
-  return 0;
-#endif
-  }
-__get_cxx_version ()
-      )";
-        auto cxx_version = Cpp::Evaluate(code);
-        return std::to_string(cxx_version);
+        std::string res = "14";
```

Comment:
Preferring 14 here (instead of 11 as per xeus-cling) cause we cover 17/20/23 and latest cppinterop defaults to 14 https://github.com/compiler-research/CppInterOp/blob/d9bf9ab7de5453d2d38af6717a4773cdd6e8d3cb/lib/CppInterOp/CppInterOp.cpp#L2941